controlpanel/frontend/jinja2/datasource-access-update.html

```diff
@@ -9,12 +9,40 @@
 {% block content %}
 <h1 class="govuk-heading-xl">{{ page_title }}</h1>
 
+{% if form.errors %}
+        <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" data-module="govuk-error-summary">
+          <h2 class="govuk-error-summary__title" id="error-summary-title">
+            There is a problem on this page
+          </h2>
+          <div class="govuk-error-summary__body">
+            <ul class="govuk-list govuk-error-summary__list">
+              <li>
+                Please see details listed below.
+              </li>
+            </ul>
+          </div>
+        </div>
+{% endif %}
+
 <section class="cpanel-section">
   <form method="post" action="{{ action_url }}">
     {{ csrf_input }}
     <input type="hidden" name="entity_type" value="{{ entity_type }}">
     <input type="hidden" name="entity_id" value="{{ entity_id }}">
-    {{ data_access_level_options(items3bucket) }}
+
+    <div {% if form.errors %}class="govuk-form-group--error"{% endif %}>
+      <fieldset class="govuk-fieldset ">
+        {% if form.errors %}
+        <span id="confirma-error" class="govuk-error-message">
```

Comment:
I'm starting to wonder if this `confirma-error` ID is not a typo now ☺️ - is it something to do with the design system maybe? 🤷‍♂️

Reply:
Nope... it's me copying and pasting typos by the looks of it... :-) Will fix.